soh/assets/soh_assets.h

```diff
@@ -57,6 +66,15 @@ static const ALIGN_ASSET(2) char gFileSelPleaseChooseAQuestFRATex[] = dgFileSelP
 #define dgFileSelPleaseChooseAQuestGERTex "__OTR__textures/title_static/gFileSelPleaseChooseAQuestGERTex"
 static const ALIGN_ASSET(2) char gFileSelPleaseChooseAQuestGERTex[] = dgFileSelPleaseChooseAQuestGERTex;
 
+#define dgFileSelBossRushSettingsENGTex "__OTR__textures/title_static/gFileSelBossRushSettingsENGTex"
+static const ALIGN_ASSET(2) char gFileSelBossRushSettingsENGText[] = dgFileSelBossRushSettingsENGTex;
+
+#define dgFileSelBossRushSettingsFRATex "__OTR__textures/title_static/gFileSelBossRushSettingsFRATex"
+static const ALIGN_ASSET(2) char gFileSelBossRushSettingsFRAText[] = dgFileSelBossRushSettingsFRATex;
+
+#define dgFileSelBossRushSettingsGERTex "__OTR__textures/title_static/gFileSelBossRushSettingsGERTex"
+static const ALIGN_ASSET(2) char gFileSelBossRushSettingsGERText[] = dgFileSelBossRushSettingsGERTex;
+
```

Comment:
Exactly. Feels weird to have to select an empty save file. It also means that you have to have an empty save file in order to play Boss Rush at all. If your save slots are all taken there's just no way to reach Boss Rush. I realize these are not trivial so we don't have to fix it now, just some rough edges to smooth out in a v2 later.

Reply:
I don't think it feels any less weird than if you had 3 rando saved and suddenly couldn't play vanilla or MQ, because you didn't have enough slots. Making it take up a save slot seems counter productive in that matter, and I just think the situation of someone having 3 saves that they don't want to delete and wanting to play boss rush is a very niche thing.

Reply:
@Patrick12115 but it does *effectively* take up a slot by needing to keep one blank. Making a boss rush "save" would set the UX groundwork for eventual future persistent stat tracking in boss rush save files etc.

Reply:
But then if you delete that save, do you lose all the stats? Because if so, you'd have to permanently have a boss rush save to keep your stats, and make sure to transfer it between updates and hope at no point an update corrupts it somehow. And if deleting that save doesn't erase the stats, then it seems like stat tracking could still be done without taking up a file slot. I get that it needs a slot to be played, but then to take up a slot afterward just doesn't make sense to me for what this is.

Reply:
stats in a single boss rush save sounds better than stats in the global save to me, i'm imagining a "bosses defeated"  challege stream where someone races to beat 100 bosses or something and starts with a fresh boss rush save to track those stats

Reply:
I don't know. This game mode is great for a lot of things, but it's more so short spurt type things, or practice. This is all just my opinion and view on this honestly, but it seems like it would be more work than it would all be worth.

Reply:
Granted I'm also not the one doing the work, so I now see how that point is kinda dumb haha

Reply:
I'm gonna vote that we keep it as is for now, I'll open an issue exploring more options after this is merged.

Reply:
There's a lot of UX questions that need to be answered to decouple boss rush from the regular saves regardless of what it ends up being. Not to mention what to do with stats if we end up wanting them. Where are they shown in the menu? How do you enter boss rush? Is there a way to not detract from the vanilla menu?

And speaking of effort required, currently it heavily builds upon the quest menu and cleans up that menu by having a guaranteed second game mode. It's not impossible to convert into something else, but I'll be honest in saying I'm not prepared to put in the effort to resolve this. 

Obviously it's fine to have a discussion for the future, and I understand having it use an empty save file is a little weird, but it was a conscience decision when I started work on this to use the flow as it is currently despite that weirdness.

Reply:
i also agree none of this should block the PR. i was thinking short/medium term post-merge

Reply:
I highly think boss rush mode requires more save slots, at very least a dedicated one, specially if you have a slot for vanilla, mq and rando, you would need to sacrifice a slot in order to play boss rush.

Reply:
This is the sort of thing where we need to get this in so it's easier for more people to mess around with and improve the UX. I'm going to mark this conversation as resolved, there will be an issue for us to discuss/workshop/demo UX changes.